### PR TITLE
Some layout/UI improvements for better usability

### PR DESCRIPTION
**Pull request by stilscher**
This PR contains several changes to improve the usability of Gobview:

- Link the setup instructions from the Read the Docs file in the README
- Have a second sidebar and imitate the layout of an editor. The left sidebar contains list of files and search tab for navigation, the right one contains the abstract state. The main/middle part is solely used for displaying either the cfg of a function or the source code of a file
- give the search result view a vertical layout to fit into the sidebar
- have a Header
- fit the cfg into the window and make it zoomable
- fix the padding and scrolling so the overall page is a one-page dashboard
- remove unecessary duplicate headings ('Result', 'Warnings')
- move some UI related files into the corresponding subfolder for better overview when developing

This PR aims at fixing the most obvious UI issues, there are still several things to improve, like better dead code highlighting in the source view, style of text/links and warnings, proper scaling when resizing a window without reloading or highlighting of cfg nodes.

Depends on #19 to actually work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Web Frontend for Goblint
 
-Originally developed by Alex Micheli for his Bachelor's thesis at TUM i2.
+The Web Frontend allows inspecting the analyzed files and results of an analysis run with Goblint.
+It is based on [jsoo-react](https://github.com/jchavarri/jsoo-react) and was originally developed by Alex Micheli for his Bachelor's thesis at TUM i2.
```
Comment (michael-schwarz) [inline README.md:4]:
```suggestion
It is based on [jsoo-react](https://github.com/jchavarri/jsoo-react) and was originally developed by Alex Micheli for his Bachelor's thesis at TUM i2, significantly extended by Kerem Cakirer, and is now maintained by the Goblint team.
```

File: src/state/gvMessages.ml [hunk 2]
```diff
@@ -10,6 +10,13 @@ module Message = struct
     | Single p -> loc p
     | Group { pieces; _ } -> pieces |> List.filter_map loc |> List.enum |> Enum.get
 
+  let severity_to_bs_alert m = match m.severity with
```
Comment (michael-schwarz) [inline src/state/gvMessages.ml:13]: Why `bs_alert`?

**Review by michael-schwarz — APPROVED** [review]
LGTM!

File: src/state/gvMessages.ml (hunk 2 above)
Comment (stilscher) [reply src/state/gvMessages.ml:13]: This function converts the severity of the warning to the string of a fitting bootstrap alert. It is used for displaying for example failures and successful warnings different colors without needing to hand-craft the css.

**Pull request merged**